index.html

```diff
@@ -218,6 +237,400 @@ <h2>
+          |manifest|["tab_strip"], and |manifest URL|.
+          </li>
+          <li>[=Process the `new_tab_button` member=] passing
+          |json|["tab_strip"], |manifest|["tab_strip"], |manifest URL|, and
+          |manifest|["start_url"].
+          </li>
+        </ol>
+        <section>
+          <h3>
+            Processing the `home_tab` member
+          </h3>
+          <p>
+            To <dfn>process the `home_tab` member</dfn>, given [=ordered map=]
+            |json:ordered map|, [=ordered map=] |manifest tab strip:ordered
+            map|, and [=URL=] |manifest URL:URL|, run the following:
+          </p>
+          <ol>
+            <li>If |json|["home_tab"] does not exist or |json|["home_tab"] not
+            an [=ordered map=], return.
+            </li>
+            <li>Let |home tab:ordered map| be a new [=ordered map=].
+            </li>
+            <li>[=Process the `scope_patterns` member=] passing
+            |json|["home_tab"]["scope_patterns"], |home tab| and |manifest
+            URL|.
+            </li>
+            <li>Set |manifest tab strip|["home_tab"] to |home tab|.
+            </li>
+          </ol>
+        </section>
+        <section>
+          <h3>
+            Processing the `new_tab_button` member
+          </h3>
+          <p>
+            To <dfn>process the `new_tab_button` member</dfn>, given [=ordered
+            map=] |json:ordered map|, [=ordered map=] |manifest tab
+            strip:ordered map|, [=URL=] |manifest URL:URL|, and [=URL=] |start
+            URL:URL|, run the following:
+          </p>
+          <ol>
+            <li>Set |manifest tab strip|["new_tab_button"]["url"] to |start
+            URL|.
+            </li>
+            <li>If |json|["new_tab_button"] does not exist or
+            |json|["new_tab_button"] is not an [=ordered map=], return.
+            </li>
+            <li>Let |url:URL| be the result of [=URL Parser|parsing=]
+            |json|["new_tab_button"]["url"] with |manifest URL| as the base
+            URL.
+            </li>
+            <li>If |url| is failure, return.
+            </li>
+            <li>If |url| is not [=URL/within scope=] of |manifest URL|, return.
+            </li>
+            <li>If |url| is [=within home tab scope=], return.
+            </li>
+            <li>Set |manifest tab strip|["new_tab_button"]["url"] to |url|.
+            </li>
+          </ol>
+          <aside class="note">
+            It is not possible to have a new tab button URL [=within home tab
+            scope=], because it would mean that activating the new tab button
+            would open the page in the home tab.
+          </aside>
+        </section>
+        <section>
+          <h3>
+            Processing the `scope_patterns` member
+          </h3>
+          <p>
+            To <dfn>process the `scope_patterns` member</dfn>, given [=ordered
+            map=] |json:ordered map|, [=ordered map=] |manifest home
```

Comment:
same? where are we in the json of the manifest?

Reply:
Done.